spec/jobs/synchronization_job_spec.rb

```diff
@@ -96,8 +96,23 @@ def performes
             end
           end
         end
-      end
 
+        context 'with sync_from' do
+          let(:opts) { { sync_from: sync_from } }
+          let(:sync_from) { Time.new(2002, 10, 31, 2, 2, 2, "+02:00") }
+
+          it { performs }
+
+          it 'passes the correct sync_from dates' do
+            organization.synchronized_entities.each do |entity, _|
+              expect_any_instance_of(Maestrano::Connector::Rails::SynchronizationJob).to receive(:sync_entity)
```

Comment:
Really just need to test the `sync_from`. Test `entity.to_s` and `organization` because their readily available. 

The other arguments don't relate to this option and would be a pain to test. Don't think its necessary.